src/Bicep.Core/TypeSystem/TypeAssignmentVisitor.cs

```diff
@@ -632,12 +688,15 @@ public override void VisitVariableAccessSyntax(VariableAccessSyntax syntax)
 
                     case VariableSymbol variable:
                         return new DeferredTypeReference(() => VisitDeclaredSymbol(syntax, variable));
+                    
+                    case NamespaceSymbol namespaceSymbol:
+                        return new UnassignableTypeSymbol(namespaceSymbol.Name);
```

Comment:
What's the reason not to follow the same pattern for `OutputSymbol`?

Reply:
A namespace is a variable syntax bound to a namespace, so I don't think it should assign an error, maybe I am missing something?